app/views/managers/requested_employees.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <table class="table mt-4 form-table table-hover mt-5">
+    <thead>
+    <tr>
+      <th scope="col" class="bg-secondary">Employee Id</th>
+      <th scope="col" class="bg-secondary">Employee Name</th>
+      <th scope="col" class="bg-secondary">Number Of Requests</th>
+    </tr>
+    </thead>
+    <tbody>
+    <% @users.each do |user| %>
+      <tr onclick="location.href='<%= show_request_manager_path(id: user.id) %>'">
+        <td><%= user.id %></td>
+        <td><%= user.name %></td>
+        <td><%= user.pending_submissions(user.id).count %></td>
```

Comment:
you are calling the function with the same user and also passing its ids doesnot make any sense